package.json

```diff
@@ -5,6 +5,7 @@
     "build:dev": "python3 ./scripts/build.py --mode dev && bash ./scripts/build.sh --mode dev",
     "serve": "npx live-server dist --port=3000 --no-browser",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test:cypress": "cypress run",
```

Comment:
are we adding this for later?

Reply:
removed because Cypress assignment moved to someone else